docs/deployment/create-account-and-orgs.md

```diff
@@ -0,0 +1,175 @@
+---
+slug: create-account-and-orgs
+append_help_link: true
+title: Create an account
+hide_title: true
+description: Create a Semgrep account and organization to prepare your deployment for the addition of repositories and team members.
+tags:
+  - Semgrep Cloud Platform
+  - Team & Enterprise Tier
+  - Deployment
+---
+
+import PlatformSigninGithub from "/src/components/procedure/_platform-signin-github.md"
+import PlatformSigninGitlab from "/src/components/procedure/_platform-signin-gitlab.md"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create a Semgrep account and set up organizations
+
+:::note Your deployment journey
+- You have gained the necessary [resource access and permissions](/deployment/checklist) required for deployment.
+:::
+
+* Add the rest of your organization (org) members to Semgrep. 
+* Configure Semgrep to scan repositories in other source code managers, such as Bitbucket.
+
+:::tip Using SSO for your initial sign-in 
+Alternatively, reach out to [<i class="fa-regular fa-envelope"></i> sales@semgrep.com](mailto:sales@semgrep.com) to set up SSO. This removes the need to sign in through a GitHub or GitLab account if you don't have one.
```

Comment:
I think we have self-service SSO now, we shouldn't need customers to do this. We could say reach out to support@semgrep.com if you have issues setting up SSO maybe?

Reply:
@sinat101 - Do we support self-service SSO **before** the user creates a Semgrep account?

Reply:
I was about to make the same comment - this is I believe specifically a flow intended for users who never log in or connect an SCM. It's still used quite commonly by the Sales Engineers from what I understand in speaking with a few of them recently.

Reply:
Ahhh ok, yes my mistake then this still makes sense. I forgot about that workflow.